kafka-ui-react-app/src/components/KsqlDb/List/List.tsx

```diff
@@ -0,0 +1,100 @@
+import Indicator from 'components/common/Dashboard/Indicator';
+import MetricsWrapper from 'components/common/Dashboard/MetricsWrapper';
+import PageLoader from 'components/common/PageLoader/PageLoader';
+import React, { FC, useEffect, useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router';
+import { fetchKsqlDbTables } from 'redux/actions/thunks/ksqlDb';
+import { getKsqlDbTables } from 'redux/reducers/ksqlDb/selectors';
+import QueryModal from 'components/KsqlDb/QueryModal/QueryModal';
+
+const headers = [
+  { Header: 'Type', accessor: 'type' },
+  { Header: 'Name', accessor: 'name' },
+  { Header: 'Topic', accessor: 'topic' },
+  { Header: 'Key Format', accessor: 'keyFormat' },
+  { Header: 'Value Format', accessor: 'valueFormat' },
+];
+
+const List: FC = () => {
+  const dispatch = useDispatch();
+  const [isModalShown, setIsShow] = useState(false);
```

Comment:
```suggestion
  const [isModalShown, setIsModalShown] = useState(false);
```

Minor. just a standard naming convention